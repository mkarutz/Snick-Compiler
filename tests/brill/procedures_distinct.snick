## All defined procedures must have distinct names

proc foo ()

    write "foo";
end

proc foo ()

    write "bar";
end

proc main ()
    foo ();
end
