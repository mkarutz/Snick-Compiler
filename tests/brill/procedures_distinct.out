Failure: Procedure with name foo redeclared.
Compilation Failed
